### PR TITLE
Improve ability to debug conntrack related issues

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: cilium/cmd/bpf_lb_list.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+var listRevNAT bool
+var serviceList = map[string][]string{}
+
+// bpfCtListCmd represents the bpf_ct_list command
+var bpfLBListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List load-balancing configuration",
+	Run: func(cmd *cobra.Command, args []string) {
+		common.RequireRootPrivilege("cilium bpf ct list")
+		if listRevNAT {
```
Comment (aalemayhu) [inline cilium/cmd/bpf_lb_list.go]: s/ct/lb/g

File: pkg/bpfdebug/debug.go [hunk 2]
```diff
@@ -131,9 +131,28 @@ func ctState(state uint32) string {
...
+var tupleFlags = map[int16]string{
+	0: "IN",
+	1: "OUT",
+	2: "RELATED",
+}
+
+func ctFlags(flags int16) string {
+	s := ""
+	for k, v := range tupleFlags {
+		if k&flags != 0 {
```
Comment (borkmann) [inline pkg/bpfdebug/debug.go:143]: Ok, given nothing precedes "IN", this masking should be fine.

**Review by borkmann — APPROVED** [review]
Apart from the small typo found by @scanf, this looks good to me.

**Review by aalemayhu — APPROVED** [review]
Looks good to me. The test failure is unrelated.

**Pull request merged**